src/managed/dir.proj

```diff
@@ -3,11 +3,15 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 
-  <Target Name="Build">
+  <Target Name="Build"
+          DependsOnTargets="GetLatestCommitHash">
     <MakeDir  Condition="!Exists('$(pOutDir)')" Directories="$(pOutDir)" />
 
     <PropertyGroup>
       <Args>/p:Configuration=$(ConfigurationGroup)</Args>
+      <_EnvironmentVariables>NUGET_PACKAGES=$(PackagesDir)</_EnvironmentVariables>
```

Comment:
Agreed.

1. I think the NUGET_PACKAGES env var can go away now.  NuGet will write the restore feeds to the assets file, so build will pick it up from there.  No need to set NUGET_PACKAGES as an env var anymore.
2. The other properties can just be passed as normal MSBUild properties `/p:`

---
In reply to: [117293682](https://github.com/dotnet/core-setup/pull/2331#discussion_r117293682) [](ancestors = 117293682)